netdata-installer.sh

```diff
@@ -755,6 +774,64 @@ else
 	run find "${NETDATA_PREFIX}/usr/libexec/netdata" -type d -exec chmod 0755 {} \;
 fi
 
+# -----------------------------------------------------------------------------
+
+install_go() {
+	# When updating this value, ensure correct checksums in packaging/go.d.checksums
+	GO_PACKAGE_VERSION="v0.0.2"
+	ARCH_MAP=(
+		'i386::386'
+		'x86_64::amd64'
+		'aarch64::arm64'
+		'armv64::arm64'
+		'armv6l::arm'
+		'armv7l::arm'
+	)
```

Comment:
Can you extend the map a bit?
I see the following from our anonymous statistics this far:

1. | x86_64 | 21,358(98.46%)
2. | armv7l | 214(0.99%)
3. | **i686** | 66(0.30%)
4. | aarch64 | 26(0.12%)
5. | armv6l | 21(0.10%)
6. | amd64 | 5(0.02%)
7. | **armv5tel** | 1(0.00%)